### ISSUE TITLE
Faster R-CNN

### ISSUE THREAD

**Issue by Victoria-Manousaki** [fpv-iplab/MECCANO#5]
Hello,

The mAP results of Faster R-CNN are computed given only one image or all the images of that segment?
Could you please elaborate?

Thank you,
Victoria

**Comment by francescoragusa** [comment]
Hello Victoria,

thanks for the interest on the MECCANO Dataset.

The mAP measure has been computed in a standard manner.
The inference of Faster-RCNN has been done on all interaction frames of each sequence. Then we calculated the mAP for each sequence (i.e. MECCANO is composed of a total of 20 sequences). 
To obtain the final mAP we averaged the 20 mAP.

I hope that I solved your issue!

**Comment by Victoria-Manousaki** [comment]
Hello Francesco,

I would like to know what you mean regarding "all interaction frames" for the case of the next active object detection. 
Which frames of the sequence contribute to the calculation of mAP for the next active object detection?


Thank you for your time!

**Comment by francescoragusa** [comment]
Sorry Victoria, I thought that you referred to the "Active Object Detection and Recognition" task.
So, for the "Next-Active Object Detection" task we calculated the mAP considering all the past frames belonging to the segments (i.e., a segment does not contain the frame in which the interaction starts but only the past frames).
We computed the COCO mAP and the mAP@50 which considers an IoU >= 0.50.

Tell me if this can help you!
